src/services/school.service.js

```diff
@@ -0,0 +1,20 @@
+const { School } = require('../models');
+
+const createSchool = async (schoolBody) => {
+  const school = await School.create(schoolBody);
+  return school;
+};
+
+const getSchools = async (subs) => {
+  return School.find({ name: { $regex: subs } });
```

Comment:
what if someone does not pass a `subs` then this request should return all schools

Reply:
so if someone input nothing, he should get nothing, right?

Reply:
so how would you suggest what to write here?

Reply:
`  if (!name || name.length === 0) {
    return School.find({ name: { $regex: name } });
  }`
can i write like this?

Reply:
```js
if (!name || name.length === 0) { 
  return School.find({} 
}
return School.find({ name: { $regex: subs } });
```

Reply:
so this still means that i get all of schools when i input nothing

Reply:
yes

Reply:
let me try

Reply:
actually i can not input nothing because it is written as required.